Cargo.toml

```
@@ -1,3 +1,5 @@
 [workspace]
-members = []
+members = [
+    "lib/core"
```

```suggestion
    "lib/*"
```

I think this is valid? Just so we can add all the other `lib/` crates later without defining them all separately

---

Oh yeah oops, I think this is just from when I made my own file in here before it was in main.